easybuild/easyblocks/n/nwchem.py

```diff
@@ -194,7 +208,11 @@ def configure_step(self):
             if self.cfg['armci_network'] in ["MPI-MT"]:
                 libmpi = "-lmpigf -lmpigi -lmpi_ilp64 -lmpi_mt"
             else:
-                libmpi = "-lmpigf -lmpigi -lmpi_ilp64 -lmpi"
+                #libmpi = "-lmpigf -lmpigi -lmpi_ilp64 -lmpi"
+                # Needed for our version of the Intel compilers, perhaps.
+                # Is there an approach that could be generalised, like consulting the output
+                # of mpiifort -show and processing it via a regular expression?
+                libmpi = "-lmpifort -lmpi -lmpigi -ldl -lrt -lpthread"
```

Comment:
@valtandor: I am not sure about this one. How would it affect the existing NWChem configs?